commons/src/main/scala/com/expedia/www/haystack/trends/commons/entities/MetricPoint.scala

```diff
@@ -54,6 +54,7 @@ object TagKeys {
   val RESULT_KEY = "result"
   val STATS_KEY = "stat"
   val ERROR_KEY = "error"
+  val INTERVAL_KEY = "interval"
```

Comment:
Should this be buried in commit about moving and pom updates?

Reply:
it was there in the aggregator metric point definitition